ev3dev2/motor.py

```diff
@@ -76,16 +76,46 @@
     raise Exception("Unsupported platform '%s'" % platform)
 
 
+class LineFollowFailed(Exception):
```

Comment:
I think we should suffix "Error" on these: https://www.python.org/dev/peps/pep-0008/#exception-names